app/routes.js

```diff
@@ -1 +1,10 @@
-exports.init = app => {};
+const users = require('./controllers/users'),
+  validation = require('./middlewares/validation'),
+  { check, validationResult } = require('express-validator/check'),
+  auth = require('./middlewares/auth');
+
+exports.init = app => {
+  app.post('/users', validation.validateResults(validation.validateSignup), users.create);
```

Comment:
you could get only via destructuring validateResults instead of always using `validation.validateResults` to make it less verbose